### ISSUE TITLE
Vortex Core support

### ISSUE THREAD

**Issue by dragonfax** [ChaoticConundrum/pok3r_re_firmware#2]
I found if I put in the right VID and PID, and tell the tool that its just a pok3r RGB, then it can communicate with the Core and even get the version number.

```
[00:00:01:601] N Version: V1.03.00
```

**Comment by ChaoticConundrum** [comment]
Great, thanks for the info! I actually looked into the the V1.04.01 firmware update released by vortex for the CORE the other day, and was able to decrypt it using the same code I wrote to decrypt the POK3R RGB firmware. Looking at the disassembly, it seems to use the same update protocol the POK3R RGB.

I have a vortex core on the way, and I already have a firmware patch ready for it, so I can get a copy of the bootloader, but I expect it will be very similar to the POK3R RGB. When I can test it, I will add support for it to pok3rtool.
